cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -8,24 +8,25 @@
 
    Creates Sidebar Breakout markup when given:
 
-   page.content:                   An object used to customize the markup.
```

Comment:
Are we not spacing these out anymore?

Reply:
I think it makes the list harder to scan, but I agree that it helps maintainability. I don't feel strongly about it.